mobile/voice_journal_app/lib/RecordingPage.dart

```diff
@@ -35,30 +46,38 @@ void displaySaved(){ //Create a little pop up letting the user know their reccor
   );
 }
 startRecording() async{
+  presently = DateTime.now();
+  formattedDateTime = DateFormat('yyy-MM-dd-HH-mm-ss').format(presently);
+  file = '$formattedDateTime.m4a';
+  controller.bitRate = 192000;
+  controller.sampleRate = 50000;
   rcrdIcon = const Icon(Icons.pause);
-  recording = true;
-  final hasPermission = await controller.checkPermission(); //get permission to use mic
-  //appDirectory = await getApplicationDocumentsDirectory();  //get the apps directory useful later
-  appDirectory = Directory('/storage/emulated/0/Download'); //Set directory for file to go to
+  appDirectory = await getApplicationDocumentsDirectory();  //get the apps directory useful later
+  //appDirectory = Directory('/storage/emulated/0/Download'); //Set directory for file to go to
   String exactDirectory = appDirectory.path;
-  presently = DateTime.now(); //Set presently string to date and time
-  String formattedDateTime = DateFormat('yyy-MM-dd-HH-mm-ss').format(presently); //Format date and time to work as a valid file name
-  path = '$exactDirectory' +'/$formattedDateTime.m4a';//Set the path to where it should go + the current date and time
-  if(hasPermission){ //If we got phone permissions
-    controller.record(path: path); //Record
-  }
+  path = '$exactDirectory' '/$file';//Set the path to where it should go + the current date and time formatting with .m4a at the end
+  recording = true; 
+  //rbox = await Hive.openBox<Recording>('recordings'); //open the box so we can put things inside of it.
+  rbox = Hive.box<Recording>('recordings');
+  currentRecording = Recording(id: '',title: formattedDateTime,audioFile: path,transcriptFile: '',timeStamp: presently,emotion:[Emotions.happy, Emotions.anger],isTranscribed: false,transcriptionId:'', duration: 0); //create the recording class
```

Comment:
let's have emotions be an empty array at the beginning, we don't want to assume the emotions.